AzureSiteReplicator.Test/AzureSiteReplicator.Test.csproj

```diff
@@ -1,105 +1,112 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It looks like you may have a line ending issue that's causing it to view all lines as having changed in this file. e.g. this can happen when you end up with a mix of CR and CR/LF in the same file, or something like that.